libs/blocks/accordion/accordion.css

```diff
@@ -23,6 +23,37 @@ dl.accordion {
   line-height: var(--type-body-s-lh);
 }
 
+.accordion-expand-all {
+  display: flex;
+  flex-wrap: wrap;
+  max-width: var(--grid-container-width);
+  margin: 0 auto 40px;
+}
+
+.accordion-expand-all button {
+  display: flex;
+  align-items: center;
+  box-sizing: content-box;
+  cursor: pointer;
+  margin-bottom: 16px;
+}
+
+.accordion-expand-all button.disabled {
+  color: #909090;
+  border-color: #909090;
+  background-color: #fff;
```

Comment:
```suggestion
  color: var(--color-gray-500);
  border-color: var(--color-gray-500);
  background-color: var(--color-white);
```

Reply:
Thanks! I added the variables for the gray. I don't see the point in having a variable for white so I kept that as is.